presets/4.3/rates/UAV_tech.txt

```diff
@@ -0,0 +1,23 @@
+#$ TITLE: UAV Tech rates
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: RATES
+#$ STATUS: OFFICIAL
+#$ KEYWORDS: UAV Tech, youtube, freestyle, HD, cinematic
+#$ AUTHOR: Mark Spatz (UAV Tech)
+#$ DESCRIPTION: 
+#$ DESCRIPTION: Freestyle rates by Mark Spatz (UAV Tech)
+#$ DESCRIPTION: 1000 degrees/sec for roll and pitch, 700 - for yaw. Very smooth around the sticks middle.
+#$ DESCRIPTION: Mark is a Betaflight contributor and tuner who prioritizes data over feelings. His YouTube channel includes some deep dives into tuning and blackbox analyzing.
```

Comment:
I’m ok with this Mark has been part of the works for a long time and is considered the gateway between BF and YouTube.